extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1126,10 +1126,9 @@ export class AmpStoryPage extends AMP.BaseElement {
     const navigationPath = this.storeService_.get(
       StateProperty.NAVIGATION_PATH
     );
-    // Navigation path is a complete list of the navigation history, including
-    // the currently active page. The previous page is the next to last element
-    // of that array.
-    const previousPageId = navigationPath[navigationPath.length - 2];
+
+    const currentPageIndex = navigationPath.lastIndexOf(this.element.id);
```

Comment:
Nit: this name is very confusing because different from the page index we have everywhere in the `amp-story.js` code

Reply:
you're right, lmk what you think of the new one